.github/workflows/build.yaml

```diff
@@ -34,8 +38,21 @@ jobs:
       - name: Run build
         run: npm run package
 
+      # Generate .vsix package for the extension
+      - name: Generate .vsix package
+        run: |
+          npm install -g @vscode/vsce
+          vsce package
+
+      - name: Upload VSIX artifact
```

Comment:
if: runner.os == 'Linux'

Reply:
So i need to upload the artifact only from linux os . Sure will incorporate the changes